.github/actions/run-tox/action.yml

```diff
@@ -0,0 +1,23 @@
+name: Run Tox
+description: Sets up tox and runs a tox environment.
+inputs:
+  tox-env:
+    description: The tox environment to execute.
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: Install Tox
+      run: |
+        python -mpip install -U pip
+        python -mpip install 'tox<=3.14.5' 'virtualenv<20.0.5'
+      shell: bash
+    - name: Execute tox -e ${{ inputs.tox-env }}
+      run: |
+        if [[ "$(uname -s)" == "Darwin" ]]; then
+          # Needed for Pillow to build successfully:
+          #   https://github.com/python-pillow/Pillow/issues/3438#issuecomment-543812237
+          export CPATH="$(xcrun --show-sdk-path)/usr/include"
```

Comment:
You <3 macOS, am I right? 😅 ...

Reply:
Yup. I was very happy to see Github changes 10x multiplier to use mac machines for private repos. Really make you think about why submit yourself to the torture if you're not developing desktop apps or phone apps. It simply makes no sense at all.